src/styles/navbar.module.css

```diff
@@ -0,0 +1,29 @@
+@import "./palette.css";
+@import url("https://fonts.googleapis.com/css?family=Roboto&display=swap");
+
+.logo {
+  width: 10rem;
+  height: 2rem;
+}
+
+.navbar {
+  background-color: var(--color1);
+  font-family: "Roboto", sans-serif;
+}
+
+.navLink {
+  color: var(--white) !important;
+  font-size: 0.9rem;
+}
+
+.navLink:hover,
+.navLinkActive {
+  border-bottom: 1px var(--white) solid;
+}
+
+@media (max-width: 768px) {
+  .navLink:hover,
+  .navLinkActive {
+    border-bottom: none;
+  }
+}
```

Comment:
would also add: 
```css
.toggler {
  position: absolute;
  top: 4px; 
  right: 0.5rem;
  border: none;
}

.toggler span {
  background-image: url("data:image/svg+xml,%3csvg viewBox='0 0 30 30' xmlns='http://www.w3.org/2000/svg'%3e%3cpath stroke='rgb(255, 255, 255)' stroke-width='1.5' stroke-linecap='round' stroke-miterlimit='10' d='M4 7h22M4 15h22M4 23h22'/%3e%3c/svg%3e") !important;
}
```

- the toggler style might not be the best approach but allows us to have the logo fully centered (in collapsed mode) and have all of the items aligned with it
- also suggest explicitly setting the hamburger menu icon to white and reducing its thickness to fit the overall design

Reply:
Could you please change these values to:
```
top: 1px; 
right: 0.1rem;
```

Reply:
Also one more thing (me just nitpicking sorry): when the navbar is collapsed could you add a .5rem top padding to the div with the first set of nav items to make all spacing between links equal